src/lib/utils/analytics.ts

```diff
@@ -76,5 +69,6 @@ export const formatSwapQuoteReceivedEventProperties = (
         : undefined,
     token_in_amount: formatToDecimal(trade.inputAmount, trade.inputAmount.currency.decimals),
     token_out_amount: formatToDecimal(trade.outputAmount, trade.outputAmount.currency.decimals),
+    minimum_output_after_slippage: trade.minimumAmountOut(allowedSlippage).toSignificant(6),
```

Comment:
maybe log the actual `allowedSlippage` here too? since there are BigNumber operations comparing outputAmount to minimumAmountOut may result in loss of precision